crates/sncast/src/starknet_commands/deploy.rs

```diff
@@ -31,7 +32,7 @@ pub struct Deploy {
     pub salt: Option<FieldElement>,
 
     /// If true, salt will be modified with an account address
-    #[clap(short, long)]
+    #[clap(long)]
     pub unique: bool,
```

Comment:
Why was this changed?

Reply:
`-u` from `unique` cannot coexist with `-u` from `url`. As `unique` is a very specialised flag, used only in one command, I've chosen to delete its short form to resolve the conflict.